.eslintignore

```diff
@@ -1,4 +1,12 @@
-_inc/client/**/test/*.js
-modules/lazy-images/js/lazy-images.js
-modules/widgets/search/js/search-widget-admin.js
-3rd-party/debug-bar/debug-bar.js
+*.min.js
+/3rd-party/debug-bar/debug-bar.js
+/_inc/blocks
+/_inc/build
+/_inc/client/**/test/*.js
```

Comment:
Hmm, this is essentially Calypso code -- running the linter on it gives a few valid warnings (padded blocks etc). Maybe remove this (`eslint --fix` can probably rid us of some warnings)?

Reply:
That line isn't new, just moved and with `/` prepended.

I removed it and ran the linter, seems like kind of a lot 😬 
Let's leave that for another PR.